spec/rails_param/param_spec.rb

```diff
@@ -66,6 +66,13 @@ def params;
           expect { controller.param! :foo, String, required: true, transform: :upcase }.to raise_error(RailsParam::InvalidParameterError, "Parameter foo is required")
         end
       end
+
+      context "when param is optional & not present" do
+        it "doesn't transform the value" do
+          allow(controller).to receive(:params).and_return({ })
+          expect { controller.param! :foo, String, required: true, transform: :upcase }.to raise_error(RailsParam::InvalidParameterError, "Parameter foo is required")
```

Comment:
Isn't this testing the same thing as the spec above it? The context says the param is optional but has `required: true`, which is confusing me.

Reply:
Yes, you're right, I didn't see the above case, now I've modified it to fit the context.

Reply:
Really sorry, I copy-pasted from the test above and forgot to make the parameter optional 🙈